cloud-sql/mysql/sqlalchemy/README.md

```diff
@@ -177,6 +177,26 @@ instance configuration.
 
 This step can be done as part of deployment but is separated for clarity.
 
+It is recommended to use the [Secret Manager integration](https://cloud.google.com/run/docs/configuring/secrets) for Cloud Run instead
+of using environment variables for the SQL configuration. The service injects the SQL credentials from
+Secret Manager at runtime via an environment variable.
+
+Create secrets via the command line:
+```sh
+echo -n "my-awesome-project:us-central1:my-cloud-sql-instance" | \
```

Comment:
does the user's project name and sql instance have to be inputted into the `my-awesome-project:us-central1:my-cloud-sql-instance` echo? If so, can we make it consistent with the other indications like `[YOUR_PROJECT_ID]`?

Reply:
Used the env var above